### PR TITLE
Refactor fhir converter container to use multiple files

### PR DESCRIPTION
**Pull request by emmastephenson**
# PULL REQUEST

## Summary
Refactors the FHIR converter to run out of three files, instead of all the functionality being kept in `main`.

**No new code here**! Just moving pieces around.

## Related Issue
No related ticket - I started working on #737 and got distracted :) 

## Additional Information
I tried to add the sample eICR request to the api docs, but ran into issues trying to use `Annotated` - guessing it's a versioning problem of some sort. I tried updating the Dockerfile, but it didn't work and I didn't want to spend too much more time going down this rabbit hole. I added the sample request anyways for devs to use as needed, and we can always go back later to read it in the API docs appropriately.

### CODE REVIEW

File: containers/fhir-converter/app/api_contract.py
Comment (emmastephenson) [inline containers/fhir-converter/app/api_contract.py:1]: Would welcome other naming suggestions here - `api_contract` was slightly smoother than `request_response_definitions`, but still feels clunky.

Comment (nickclyde) [reply containers/fhir-converter/app/api_contract.py:1]: Hmm, maybe this could just be `constants`?

**Review by nickclyde — APPROVED** [review]
Looks good! Nice refactor

File: containers/fhir-converter/app/api_contract.py
Comment (emmastephenson) [reply containers/fhir-converter/app/api_contract.py:1]: I like that! Renamed

**Review by BradySkylight — APPROVED** [review]
LGTM!  And the addition of the FHIR Conversion input example/request is great.

**Pull request merged**